File: beagle-ui/src/main/java/br/com/zup/beagleui/framework/widget/ui/SelectView.kt
package br.com.zup.beagleui.framework.widget.ui

import br.com.zup.beagleui.framework.widget.core.Widget

data class SelectView (
    val rows: Widget? = null,
    val remoteDataSource: String? = null,
    val loadingState: Widget? = null
) : Widget

File: Sources/BeagleUI/Sources/Widgets/UIComponents/SelectView.swift
//
//  SelectView.swift
//  BeagleUI
//
//  Created by Daniel Tes on 12/09/19.
//  Copyright © 2019 Daniel Tes. All rights reserved.
//

public struct SelectView: Widget {
    let rows: [Widget]
    let remoteDataSource: String?
    let loadingState: Widget?
}
